main.tf

```diff
@@ -12,27 +12,38 @@ locals {
 }
 
 module "task_label" {
-  source     = "cloudposse/label/null"
-  version    = "0.25.0"
-  enabled    = local.create_task_role
-  attributes = ["task"]
+  source      = "cloudposse/label/null"
+  version     = "0.25.0"
+  enabled     = local.create_task_role
+  attributes  = ["task"]
+  label_order = var.label_order_custom.task
 
   context = module.this.context
 }
 
 module "service_label" {
-  source     = "cloudposse/label/null"
-  version    = "0.25.0"
-  attributes = ["service"]
+  source      = "cloudposse/label/null"
+  version     = "0.25.0"
+  attributes  = ["service"]
+  label_order = var.label_order_custom.service
 
   context = module.this.context
 }
 
 module "exec_label" {
-  source     = "cloudposse/label/null"
-  version    = "0.25.0"
-  enabled    = local.create_exec_role
-  attributes = ["exec"]
+  source      = "cloudposse/label/null"
+  version     = "0.25.0"
+  enabled     = local.create_exec_role
+  attributes  = ["exec"]
+  label_order = var.label_order_custom.exec
```

Comment:
why not `var.iam_label_order`?

Reply:
As we've discussed in the meeting, i'm fine with a `label_orders` object with properties per service.